common/reviews/api/rush-sdk.api.md

```diff
@@ -0,0 +1,39 @@
+## API Report File for "@rushstack/rush-sdk"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+// @public
+export interface ILoadSdkAsyncOptions {
+    abortSignal?: AbortSignal;
+    onNotifyEvent?: SdkNotifyEventCallback;
+    rushJsonSearchFolder?: string;
+}
+
+// @public
+export interface IProgressBarCallbackLogMessage {
+    kind: 'info' | 'debug';
+    text: string;
+}
+
+// @public
+export interface ISdkCallbackEvent {
+    logMessage: IProgressBarCallbackLogMessage | undefined;
+    progressBarPercent: number | undefined;
+}
+
+// @public
+export class RushSdkLoader {
+    static get alreadyLoaded(): boolean;
+    static loadAsync(options?: ILoadSdkAsyncOptions): Promise<void>;
```

Comment:
Could just have this immediately resolve if it's already loaded.

Reply:
It was initially designed that way, but I think it's counterintuitive in this case:

**otherFile.ts**
```ts
// This causes rush-lib to get loaded from the cwd
import { RushConfiguration } from '@rushstack/rush-sdk';
```

**index.ts**
```ts
import { Something } from './otherFile';

// Promise resolves successfully, but rush-lib is not loaded from the requested folder
await RushSdkLoader.loadAsync({ rushJsonSearchFolder: "path/to/my-repo/apps/my-app" });
```